gdx/build.gradle

```diff
@@ -42,3 +38,10 @@ test {
         events "passed", "skipped", "failed", "standardOut", "standardError"
     }
 }
+
+// Workaround needed for IDEA to have resources on classpath when running tests (like gdx-tests-lwjgl3)
+task copyIdeaResources(type: Copy) {
+    from "${projectDir}/src/main/resources"
```

Comment:
Should probably be `${projectDir}/res` for libgdx since it still uses  non-standard folders.

Reply:
thanks, your're right, fixed